hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/Util.java

```diff
@@ -10,6 +10,7 @@
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Optional;
 import java.util.Set;
 
 public final class Util {
```

Comment:
This Util class which is in ca.bc.gov.hlth.hnsecure.parsing seems to cover a lot more than just parsing related utilities now. Not sure what a better package would be though. Something to think about.

Reply:
yeah I agree, Util is getting bigger with each feature. Need to segregate in multiple classes.